ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -590,7 +590,7 @@ export PATH=${OC_DIR}:$PATH
 for target in "${TARGET_RELEASES[@]}"
 do
     export TARGET="${target}"
-    TARGET_VERSION="$(env "NO_PROXY=*" "no_proxy=*" oc adm release info "${TARGET}" -o jsonpath='{.metadata.version}')"
+    TARGET_VERSION="$(env "NO_PROXY=*" "no_proxy=*" oc adm release info "${TARGET}" --output=json | jq -r '.metadata.version')"
```

Comment:
wondering what's the correct oc version here, 4.1?

Reply:
The `oc` is extracted from the `latest` image. In upgrade test, it's the source version.